CMakeLists.txt

```diff
@@ -370,14 +370,10 @@ else()
     set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-z noexecstack -z relro -z now -Wl,--gc-sections")
 endif()
 
-if (ISPC_STATIC_STDCXX_LINK)
+if (ISPC_STATIC_STDCXX_LINK OR ISPC_STATIC_LINK)
     set_target_properties(${PROJECT_NAME} PROPERTIES LINK_FLAGS "-static-libgcc -static-libstdc++")
```

Comment:
Here set_target_properties setting will rewrite LINK_FLAGS, not append to them. Consider using target_link_options instead.